exercise/emailType.js

```diff
@@ -0,0 +1,40 @@
+/*
+## Email Type
```

Comment:
I think emails are a bit risky to teach to be parsed with regex as this is the source of many comon errors in programing.

Turn's out matching **valid** emails is really tricky (see https://emailregex.com/)
I would not recommand anyone to use regex for that.

So what could we do ?
Maybe just extract the domain of an email ?

I'm not sure I would go near emails with regexes tbh, if I have to match an email I would just do:
`/(\S+@\S+)/` because other than that it's too hard to make sure you won't handle any wierd email that is completly valid.

So, i'm against this exercises because it encourages bad practicies, email + regexp = bad mix.